code/modules/chemistry/Reagents-Misc.dm

```diff
@@ -1569,8 +1569,11 @@ datum
 
 			reaction_mob(var/mob/M, var/method=TOUCH, var/volume)
 				. = ..()
+				if (method == TOUCH)
+					. = 0 // for depleting fluid pools
```

Comment:
Not a great solution for scalability; I think some fundamental parts of how reagents are handled need to be re-designed wrt what constitutes a "proper" reaction or not, but I'd rather not poke around in there for such a simple change and potentially break a ton of reagents and interactions outright.